chinese_tutorial_rvest_httr.Rmd

```diff
@@ -0,0 +1,490 @@
+
+#Chinese Web scraping using R package: rvest & httr tutorial
+
+Huiqian Yu
+
+```{r install, eval=FALSE, include=FALSE}
+install.packages("rvest") 
+install.packages("httr")
+```
+
+```{r global_options, include=FALSE}
+library("rvest")
+library("httr")
+```
+
+## Summary:
+
+This is a tutorial in Chinese shows how to implement  `rvest` and `httr` package for web scraping. 
+
+This note covers fundamental package description, four examples on web scraping: 1. scrap R package infomation on CRAN, 2. scrap R related question on stack overflow  using `rvest` package and scrap top 250 movies from dynamic website using  `httr` package, which users could simply practice and start their own scrapper. I choose not to repeat html, css and xpath in this note since there are former work cover these. 
+
+
+
+## 1. 使用 rvest 包抓取数据
```

Comment:
Hashtag headers should not be followed by numbers since the hashtags will create numbered subheadings

Reply:
This problem has been solved, thank you.